doc/tutorials/cf/cf.txt

```diff
@@ -310,7 +310,7 @@ factorization with alternating least squares update rules).  These include:
 
 The amf::AMF<> class has many other possibilities than those listed here; it is
 a framework for alternating matrix factorization techniques.  See the
-\ref amf::AMF<> "class documentation" or \ref amftutorial "tutorial on AMF" for
+\ref amf.hpp "class documentation" or \ref amftutorial "tutorial on AMF" for
```

Comment:
Hmm, the idea before was to link to the class documentation for `amf::AMF<...>`.  This links to the file instead though.  Does `amf::AMF` work?

Reply:
No, it's not too happy about that either 
```
Generating docs for page cftutorial...
/home/ryan/Documents/mlpack/doc/tutorials/cf/cf.txt:319: error: unable to resolve reference to `amf::AMF' for \ref command (warning treated as error, aborting now)
Exiting...
CMakeFiles/doc.dir/build.make:57: recipe for target 'CMakeFiles/doc' failed
make[3]: *** [CMakeFiles/doc] Error 1
```

Reply:
Ah, I think it wanted " mlpack::amf::AMF<>"; changing.

Reply:
Awesome! :rocket: